scripts/shared.sh

```diff
@@ -32,6 +32,7 @@ function parse_std_args() {
     # defaults
     arch='cpu'
     version='1.12.0'
+    model='AmazonEI_TensorFlow_Serving_v1.12_v1'
```

Comment:
The file hosted in S3 is a zip, where does that get handled?

Reply:
also 'model' isn't a good name

Reply:
+1 the s3 fetch is missing, and there should be updates to the build.sh as well (or nothing is getting built).

also, if the S3 fetch normalized the file name when it downloaded, then there would likely be no need to pass the filename into the dockerfile as an ARG.